aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -611,7 +612,11 @@ Resources:
       IamInstanceProfile: !Ref DaemonInstanceProfile
       KeyName: <%=SSH_KEY_NAME%>
       Tags:
-        - {Key: Name, Value: <%=frontends ? "#{environment}-daemon" : "!Ref 'AWS::StackName'"%>}
+<%   if frontends -%>
+        - {Key: Name, Value: !Sub "${AWS::StackName}-daemon"}
+<%   else -%>
+        - {Key: Name, Value: !Ref AWS::StackName}
+<%   end -%>
```

Comment:
I believe this would change `production-daemon` to be renamed `autoscale-prod-daemon`.